Resources/Prototypes/Entities/Structures/plastic_flaps.yml

```
@@ -97,19 +99,25 @@
 
 - type: entity
   id: PlasticFlapsAirtightOpaque
-  parent: PlasticFlapsBase
+  parent: PlasticFlapsAirtightClear
```

This should be PlasticFlapsOpaque
```suggestion
  parent: PlasticFlapsOpaque
```

---

PlasticFlapsOpaque has a construction graph, how would I disable the graph in the child? I attempted to do it but I only came up with errors.